backends/backends.c

```diff
@@ -507,6 +514,7 @@ void *backends_main(void *ptr) {
     const char *destination     = config_get(CONFIG_SECTION_BACKEND, "destination", "localhost");
     global_backend_prefix       = config_get(CONFIG_SECTION_BACKEND, "prefix", "netdata");
     const char *hostname        = config_get(CONFIG_SECTION_BACKEND, "hostname", localhost->hostname);
+    const char *remote_hostname = config_get(CONFIG_SECTION_BACKEND, "remote hostname", "localhost");
```

Comment:
The `remote hostname` option isn't needed. You should use the `destination` instead.